torch/csrc/jit/mobile/backport.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <istream>
+#include <memory>
+
+namespace caffe2 {
+namespace serialize {
+class ReadAdapterInterface;
+class PyTorchStreamWriter;
+} // namespace serialize
+} // namespace caffe2
+
+namespace torch {
+namespace jit {
+
+// The family of methods below load a serialized Mobile Module
```

Comment:
Overall I don't think it's worth having a public API for backport to -1 along with backport to a specific version.